src/components/ConfirmPublishAlert/index.jsx

```diff
@@ -53,7 +53,10 @@ export default function ConfirmPublishAlert({
           <Button
             value="Publish"
             variant="solid"
-            onClick={publish}
+            onClick={async () => {
```

Comment:
Once we bake in some async / loading UI this can probably be lifted to a proper function